autoload/vital/__vital__/App/WindowSelector.vim

```diff
@@ -5,6 +5,7 @@ function! s:select(winnrs, ...) abort
         \ 'statusline_hl': 'VitalWindowSelectorStatusLine',
         \ 'indicator_hl': 'VitalWindowSelectorIndicator',
         \ 'use_popup': 0,
+        \ 'popup_borderchars': has('nvim') ? ['╭', '─', '╮', '│', '╯', '─', '╰', '│'] : ['─', '│', '─', '│', '╭', '╮', '╯', '╰'],
```

Comment:
We need to normalize border chars in this module so that developers don't need to care about the difference between Vim and Neovim.

Reply:
Done